impeller/renderer/backend/gles/gles.h

```diff
@@ -4,6 +4,8 @@
 
 #pragma once
 
+// IWYU pragma: begin_exports
```

Comment:
This is neat, apparently you can tell IWYU how includes are being used:
https://github.com/include-what-you-use/include-what-you-use/blob/master/docs/IWYUPragmas.md#iwyu-pragma-export